stubs/core.stub

```diff
@@ -1,5 +1,12 @@
 <?php
 
+/**
```

Comment:
stub taken 1:1 from psalm

Reply:
udpate: psalm was missing the type for $allow_string, which will be added with https://github.com/vimeo/psalm/pull/7951